howto/install-appliance/google-cloud.md

```diff
@@ -0,0 +1,107 @@
+
+## Prerequisites
+
+Before starting the procedure,
+* Check the hardware requirements for the Anbox Cloud Appliance [here](https://discourse.ubuntu.com/t/17734).
+* Make sure that you have a Google Cloud account and a project on Google Cloud to create the virtual machine.
+* If you wish to use your own Ubuntu Pro subscription, ensure you have the Ubuntu Pro token for your Ubuntu Pro subscription. If you wish to use the Ubuntu Pro subscription offered by Google Cloud along with the virtual machine, skip this prerequisite.
+
+## Virtual machine setup
+
+### 1. Create a virtual machine
+Log in to [Google Cloud](https://console.cloud.google.com) and select the project for this deployment. Select **CREATE INSTANCE** in the **VM instances** page.
+
+![VM instances on Google Cloud|690x440](https://assets.ubuntu.com/v1/d44bd0dd-VM-instance.png)
+
+Select **New VM instance** from the options on the sidebar.
+
+![VM instance options on Google Cloud|690x440](https://assets.ubuntu.com/v1/cbc94514-Instance-options.png)
+
+### 2. Configure basic settings
+
+Configure the following basic settings for the virtual machine. 
+
+* **Name** - Name of the virtual machine instance
+* **Labels** - Organisational labels to keep track of your resources on Google Cloud
+* **Region** and **Zone** - The geographic location where your resources are run and your data is stored
+* **Machine configuration**, **Series**, **Machine type** and **Display device** - Select a machine configuration that matches the hardware requirements. If your requirement includes GPUs, select the GPU-optimised machine configuration. For example, an NVIDIA L4 GPU. You can select the recommended preset machine type or define a custom type with additional cores and memory for an added cost.
+
+Most of the configuration depends on your deployment and its location. For the settings that are not mentioned in this guide, you can choose to proceed with the default options or see [Google documentation](https://cloud.google.com/compute/docs/instances/create-start-instance) to customise your virtual machine on Google Cloud.
+
+### 3. Configure disks
+
+Configure the boot disk by selecting **Boot disk > CHANGE** to choose the operating system and boot disk size.
+
+![Configure boot disk|690x440](https://assets.ubuntu.com/v1/d85bed8c-boot-disk-config.png)
+
+Select the operating system. Google Cloud has two options - Ubuntu and Ubuntu Pro. If you have an Ubuntu Pro subscription already, you can choose Ubuntu and [attach your subscription](https://discourse.ubuntu.com/t/22681#attach-ubuntu-pro) manually. If you don’t, you can choose the Ubuntu Pro option which will include a Ubuntu Pro subscription through Google Cloud. 
+
+[note type="information" status="Note"]Remember that choosing the Ubuntu Pro option will have an impact on your pricing. The **Pricing summary** section provides the resource details and the associated costs.[/note]
```

Comment:
```suggestion
[note type="information" status="Note"]Remember that choosing the Ubuntu Pro option will impact your pricing. The **Pricing summary** section provides the resource details and the associated costs.[/note]
```
I feel like it reads a bit easier, but that may just be personal preference. Feel free to ignore if you disagree!

Another thing: the Pricing summary is on the GCP website, right? Reading this the first time, I kinda expected a documentation section with the Pricing summary. That may be just because I am unfamiliar with GCP, but I felt a bit confused when I didn't find it. Maybe adding that the section is on the GCP page could help? e.g.:
```suggestion
[note type="information" status="Note"]Remember that choosing the Ubuntu Pro option will have an impact on your pricing. Google Cloud provides the resource details and the associated costs in the **Pricing summary** section.[/note]
```
What do you think?